src/frontend/src/molecules/AuthButton.jsx

```diff
@@ -31,20 +31,22 @@ const AuthButton = () => {
     }
   };
 
+  const btn = "btn btn-outline-secondary m-1"
+
   return (
     <div>
       {modalPage && <AuthModal page={modalPage} onClose={() => setModalPage(null)} />}
       {user && (
-        <a href="#" onClick={() => signOut()}>
+        <a className={btn} href="#" onClick={() => signOut()}>
```

Comment:
Thanks for the PR, Matt! Can you use `Button.jsx` instead?

Reply:
Hopefully this is more in-line with the styling for the platform.